### ISSUE TITLE
Incorporate plot_ly() config arguments ?

### ISSUE THREAD

**Issue by akarlinsky** [erblast/parcats#2]
After playing with my alluvial plot, I would like to save it as a png. But the resulting image is very small (700 x 450). I would have liked to use plot_ly() config arguments (https://plotly-r.com/exporting-static-images.html) such as:
`plot_ly() %>%
  config(
    toImageButtonOptions = list(
      format = "svg",
      filename = "myplot",
      width = 600,
      height = 700
    )
  )`

But these have no effect on a paracats object... Could this be incorporated or an option to set the size of exports of paracts objects directly?

**Comment by erblast** [comment]
yes so this does not work, because it is a different package, but you can use the the Rstudio export button in the viewer window to create any screenshot in a reasonable resolution that you like.
If you are looking for an automated solution you can use `the package `webshot` which integrates well with `knitr`. You can find a detailed description [here](https://bookdown.org/yihui/bookdown/html-widgets.html).  If at one point the R plotly package will include the parcats trace, I might add a a wrapper for it to `easyalluvial`. But I do not think this would add much additional functionality besides it being easier to use for people that already know the plotly R package API.
